CHANGELOG.md

```diff
@@ -5,9 +5,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `SubspaceDiscrete.to_searchspace` and `SubspaceContinuous.to_searchspace`
+  convenience constructor
+
 ### Changed
 - Passing an `Objective` to `Campaign` is now optional
 
+### Fixed
+- `sequential` flag of `SequentialGreedyRecommender` is now set to `True`
```

Comment:
@Scienfitz: OK, PR ready to go 🚀  Last question: are we now happy with `BotorchRecommender`? I made my peace with it, because it is the most accurate name I saw until now, but also want your final approval 🙃

Reply:
sure